components/quiz/Quiz.tsx

```diff
@@ -12,10 +13,25 @@ export type Answer = {
 };
 
 function Quiz({ question, answerOptions }: QuizProps) {
+  const handleAnswerClick = (isCorrect: boolean) => {
+    if (isCorrect === true) {
+      alert("AMAZING"), setColorchange("green");
+    } else {
+      alert("Hmm, try again!"), setColorchange("red");
+    }
+  };
+  const [colorchange, setColorchange] = useState("blue");
```

Comment:
It's possible to define the state in this line, but for a consistent code style, I would move states and other hooks to the top of this component/function.

Reply:
`camelCase` would be `colorChange`

Reply:
Instead of using colors in this state, I would prefer a more meaningful value like:

```ts
const [guessStatus, setGuessStatus] = useState<'pending' | 'wrong' | 'correct'>('pending');
```

Feel free to outsource the union type:
```ts
type AnswerStatus = 'pending' | 'wrong' | 'correct';
```

Color the button `green` if it is correct is a styling question. Maybe you like to change the style in the future and use `lightgreen` or `#2ea043` as color. This shouldn't require a (js) code change. Instead, I would create CSS classes for `.wrong` and `.correct` answers, which colorize your buttons.

Reply:
> Instead of using colors in this state, I would prefer a more meaningful value like:
> 
> ```ts
> const [guessStatus, setGuessStatus] = useState<'pending' | 'wrong' | 'correct'>('pending');
> ```
> 
> Feel free to outsource the union type:
> 
> ```ts
> type AnswerStatus = 'pending' | 'wrong' | 'correct';
> ```
> 
> Color the button `green` if it is correct is a styling question. Maybe you like to change the style in the future and use `lightgreen` or `#2ea043` as color. This shouldn't require a (js) code change. Instead, I would create CSS classes for `.wrong` and `.correct` answers, which colorize your buttons.

classNames are set to Quizbutton in CSS, I would like that every button is united with these CSS class and the boolean value, but don't know how I write right CSS  statement in if statement.

Reply:
Take a look at your button component or use a lib like
https://www.npmjs.com/package/classnames